.gitignore

```diff
@@ -49,6 +49,7 @@ typings/
 
 # Optional npm cache directory
 .npm
+yarn.lock
```

Comment:
Please remove this. You shouldn't be editing the .gitignore file.

Reply:
if i leave yarn.lock it will cause issue for those who are using npm not yarn